<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>1018</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>1311</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/image/picter/kvartili-vak-2.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="fillDatabaseButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>600</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть заполнение</string>
    </property>
   </widget>
   <widget class="QComboBox" name="columnSelector">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>290</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>критерии поиска </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Наименование издания </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> ISSN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Направление</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дата включения</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>323</y>
      <width>851</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchInput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>611</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="resultsTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>1281</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetResults">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>1281</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadTxtButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>891</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>загрузить File txt </string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>600</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть заполнение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CleanButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>650</y>
      <width>291</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить Бзу данных</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
